ExampleSwiftUI/SwiftAudioPlayer-SwiftUI/SwiftAudioPlayer-SwiftUI/ContentView.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  ContentView.swift
+//  SwiftAudioPlayer-SwiftUI
+//
+//  Created by Ran on 4/21/21.
+//
+
+import SwiftUI
+
+struct ContentView: View {
+    @StateObject var saUpdates = SAPlayer.SAPlayerCombine.shared
```

Comment:
If you're going to use saUpdates in multiple views, consider moving this to your `App` file and inject it into each view. For each view save it as `@ObservedObject` instead of `@StateObject`